cmd/checkcommits/checkcommits_test.go

```diff
@@ -642,7 +642,7 @@ func TestGetCommitAndBranch(t *testing.T) {
 		}
 
 		// XXX: crucially, no arguments (to trigger the auto-detection)
-		commit, dstBranch, err := getCommitAndBranch([]string{}, []string{})
+		commit, dstBranch, _ := getCommitAndBranch([]string{}, []string{})
```

Comment:
Sorry, just noticed this: this is an oversight - we should be retrieving the error, then:

```
if err != nil {
    t.Fatal(err)
}
```